<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialogSettings</class>
 <widget class="QDialog" name="QDialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1" colspan="3">
      <widget class="QLineEdit" name="logFolder"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>204</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Camera save folder location:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>204</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Trend buffer size[mm:ss]:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="searchLog">
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/icons/open.png</normaloff>:/images/icons/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="simFile"/>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="forceScene">
       <property name="text">
        <string>Sensor graphics ON on startup</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>204</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Log save folder location:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="searchCam">
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/icons/open.png</normaloff>:/images/icons/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="arenaFolder"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>204</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Simulator location:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="searchArena">
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/icons/open.png</normaloff>:/images/icons/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>204</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Trend sample time[s]:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>204</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Session save folder location:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTimeEdit" name="trendTimeSpan">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximumTime">
        <time>
         <hour>0</hour>
         <minute>59</minute>
         <second>59</second>
        </time>
       </property>
       <property name="displayFormat">
        <string>mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="camFolder"/>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="forceLog">
       <property name="text">
        <string>Logging ON on startup</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="searchSim">
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/icons/open.png</normaloff>:/images/icons/open.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="trendSampleTime">
       <property name="prefix">
        <string/>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>9.900000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QDialogSettings</receiver>
   <slot>saveConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QDialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchLog</sender>
   <signal>clicked()</signal>
   <receiver>QDialogSettings</receiver>
   <slot>findFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchCam</sender>
   <signal>clicked()</signal>
   <receiver>QDialogSettings</receiver>
   <slot>findFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchArena</sender>
   <signal>clicked()</signal>
   <receiver>QDialogSettings</receiver>
   <slot>findFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchSim</sender>
   <signal>clicked()</signal>
   <receiver>QDialogSettings</receiver>
   <slot>findFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findFolder()</slot>
  <slot>saveConfig()</slot>
 </slots>
</ui>
